cs/test/core/Core.csproj

```diff
@@ -29,53 +23,23 @@
     <BondOptions>--fields --collection-interfaces</BondOptions>
   </PropertyGroup>
   <ItemGroup>
-    <BondImportDirectory Include="import dir with spaces\" />
+    <BondImportDirectory Include="import dir with spaces" />
```

Comment:
You had mentioned that when this had the trailing slash you were getting a warning. I cannot repro that warning with MSBuild 15.5.180.51428 and .NET Core SDK 2.1.4.

Perhaps it was showing up as an artifact of something else that hadn't been fixed during the migration.
If we can leave the trailing slash, let's do that.

Reply:
Gotcha. It's from the Visual Studio project system. Try this patch:

```patch
diff --git a/cs/build/nuget/Common.targets b/cs/build/nuget/Common.targets
index 70f6d268..8ae0ed45 100644
--- a/cs/build/nuget/Common.targets
+++ b/cs/build/nuget/Common.targets
@@ -52,7 +52,7 @@
       <Context>File;BrowseObject;</Context>
     </PropertyPageSchema>
     <PropertyPageSchema Include="$(MSBuildThisFileDirectory)cps\BondImportDirectory.xaml">
-      <Context>File;BrowseObject;</Context>
+      <Context>Invisible</Context>
     </PropertyPageSchema>
   </ItemGroup>
```

Reply:
That worked